openassessment/templates/openassessmentblock/oa_base.html

```diff
@@ -3,7 +3,7 @@
 <div class="wrapper wrapper--xblock wrapper--openassessment theme--basic">
     <div class="openassessment problem" id="openassessment">
         <div class="wrapper--grid">
-            <h2 class="openassessment__title problem__header">{{ title }}</h2>
+            <h2 class="openassessment__title problem__header">{% trans title %}</h2>
```

Comment:
Perhaps we are better off just not attempting to translate the default title. Since we expect course authors to replace this default title with something that will then not be translated, there is very little benefit in translating the default title.